src/System.Management.Automation/engine/LanguagePrimitives.cs

```diff
@@ -1502,6 +1502,13 @@ internal static object GetConverter(Type type, TypeTable backupTypeTable)
                 return NewConverterInstance(assemblyQualifiedtypeName);
```

Comment:
@iSazonov, your last commit had 1 failures in `PowerShell-CI-macos`
Language Primitive Tests.Convertion with 'TypeConverterAttribute' should be before convertion with .Net Core intrinsic type convertor
```powershell
Expected '[System.Management.Automation.PSMemberTypes]::Methods', but got Methods.
at <ScriptBlock>, /Users/runner/runners/2.158.0/work/1/s/test/powershell/engine/Api/LanguagePrimitive.Tests.ps1: line 46
46:         $result | Should -Be [System.Management.Automation.PSMemberTypes]::Methods
```